test/yes-app/index.js

```diff
@@ -5,11 +5,12 @@ var explitive = process.argv.length > 2
 var i = setInterval(console.log.bind(console, explitive), 500);
 
 process.on('internalMessage', function(msg) {
-  if(msg && msg.cmd && msg.cmd === 'NODE_CLUSTER_disconnect') {
+  if (msg && msg.cmd && msg.cmd === 'NODE_CLUSTER_disconnect') {
     // v0.10
     clearInterval(i);
   }
-  if(msg && msg.cmd && msg.cmd === 'NODE_CLUSTER' && msg.act === 'disconnect') {
+  if (msg && msg.cmd && msg.cmd === 'NODE_CLUSTER'
```

Comment:
I think the `msg.cmd` in `msg && msg.cmd && msg.cmd === 'X'` is redundant and removing it might shorten these lines enough to avoid breaking it.